test/fixtures/api/graphql/branch-ref-nodes.json

```diff
@@ -13,7 +13,7 @@
                   }
                 ]
               },
-              "name": "TES-321-branch-name",
+              "name": "test-branch-name",
```

Comment:
I wonder in which tests it is used? I am afraid it can affect some tests.

(/nit Should we name it explicitly `branch-with-issue-key-in-the-last-commit` to improve tests readability?)